xline/src/storage/revision.rs

```diff
@@ -69,3 +81,18 @@ impl KeyRevision {
         Revision::new(self.mod_revision, self.sub_revision)
     }
 }
+
+#[cfg(test)]
+mod test {
+    use prost::bytes::Buf;
+
+    use super::*;
+    #[test]
+    fn it_works() {
```

Comment:
What does the `it` refer to in `it_works`? Please modify the test name to make it more meaningful. You can rename it as `encode_should_work` or something like that.

Reply:
fixed